frontend/app/services/EventbriteService.scala

```diff
@@ -153,6 +144,33 @@ object GuardianLiveEventService extends EventbriteService {
   }
 }
 
+object DiscoverEventService extends EventbriteService {
+  val apiToken = Config.eventbriteDiscoverApiToken
+  val maxDiscountQuantityAvailable = 2 //TODO are these discounts correct for discovery?
+
+  val wsMetrics = new EventbriteMetrics("Discover")
+
+  val gridService = GridService(Config.gridConfig.url)
+  val contentApiService = GuardianContentService
+
+  def mkRichEvent(event: EBEvent): Future[RichEvent] = {
+
+    val eventbriteContent = contentApiService.content(event.id)
+
+    event.mainImageUrl.fold(Future.successful(DiscoverEvent(event, None, eventbriteContent))) { url =>
+      gridService.getRequestedCrop(url).map(DiscoverEvent(event, _, eventbriteContent))
```

Comment:
Are Discover events going to be using the Grid? Fine if it is but if not we could remove the getRequestCrop method.